db/migrate/20200723124836_add_uniqueness_to_downloaded_files_local_path.rb

```diff
@@ -0,0 +1,19 @@
+class AddUniquenessToDownloadedFilesLocalPath < ActiveRecord::Migration[5.2]
+  def change
+    # Remove duplicates before adding uniqueness to `local_path`
+    ActiveRecord::Base.connection.execute(
+      <<~SQL
+        DELETE FROM `downloaded_files`
```

Comment:
I tested it locally with a few scenarios and on another branch to be sure it would not break the pipeline, it works fine. Added your suggestion in 72bfe76.